tests/e2e/map/map_spatial.spec.js

```diff
@@ -622,10 +622,10 @@ test.describe('Map: Spatial interactions', () => {
           await spatialDropdownMenu.getByRole('button', { name: 'Rectangle' }).click()
 
           // Enter the bounding box values
-          await page.getByTestId('spatial-display_southwest-point').focus()
-          await page.keyboard.type('-9.53964,42.1875')
-          await page.getByTestId('spatial-display_northeast-point').focus()
-          await page.keyboard.type('4.5297,56.25')
+          await page.getByTestId('spatial-display_southwest-point').click()
+          await page.getByTestId('spatial-display_southwest-point').fill('-9.53964,42.1875')
```

Comment:
The only issue with the .fill instead of type is that it implies for the test users are going to paste in that value from clipboard instead of type it out kind of think we should stick with the existing. I assume this was to try to see if we could cut out an intermittently failing test